libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -179,11 +180,24 @@ func (m *Manager) GetStats() (*cgroups.Stats, error) {
 }
 
 func (m *Manager) Set(container *configs.Config) error {
-	for name, path := range m.Paths {
-		sys, err := subsystems.Get(name)
-		if err == errSubsystemDoesNotExist || !cgroups.PathExists(path) {
+	for _, sys := range subsystems {
+		// We can't set this here, because after being applied, memcg doesn't
+		// allow a non-empty cgroup from having its limits changed.
+		if sys.Name() == "memory" {
```

Comment:
I don't think we can make this change now or in this pr because updating memory limits, not kernel limits, is a big usecase and this just stops it.  Maybe can we keep the changes small and only do the pid change in this pr and update the other things later?